### PR TITLE
Fixup init-tools-native write-verbose so that arguments are printed to console

### PR DESCRIPTION
**Pull request by jonfortescue**

### CODE REVIEW

File: eng/common/init-tools-native.ps1 [hunk 1]
```diff
@@ -98,7 +98,7 @@ try {
       }
 
       Write-Verbose "Installing $ToolName version $ToolVersion"
-      Write-Verbose "Executing '$InstallerPath $LocalInstallerArguments'"
+      Write-Verbose "Executing '$InstallerPath $($LocalInstallerArguments.Keys.ForEach({"$_ $($LocalInstallerArguments.$_)"}) -join ' ')'"
```
Comment (alexperovich) [inline eng/common/init-tools-native.ps1]: This needs the '-' before each key to look like the actual argument list.
```suggestion
      Write-Verbose "Executing '$InstallerPath $($LocalInstallerArguments.Keys.ForEach({"-$_ $($LocalInstallerArguments.$_)"}) -join ' ')'"
```

Comment (alexperovich) [reply eng/common/init-tools-native.ps1]: The value probably also needs to be quoted

Comment (jonfortescue) [reply eng/common/init-tools-native.ps1]: Ah good points. I will add those changes and merge

File: eng/common/init-tools-native.ps1 [hunk 2]
```diff
@@ -98,7 +98,7 @@ try {
       }
 
       Write-Verbose "Installing $ToolName version $ToolVersion"
-      Write-Verbose "Executing '$InstallerPath $LocalInstallerArguments'"
+      Write-Verbose "Executing '$InstallerPath $($LocalInstallerArguments.Keys.ForEach({"-$_ `"$($LocalInstallerArguments.$_)`""}) -join ' ')'"
```
Comment (vatsan-madhavan) [inline eng/common/init-tools-native.ps1]: This is failing builds. See https://dev.azure.com/dnceng/internal/_build/results?buildId=276755

Comment (jonfortescue) [reply eng/common/init-tools-native.ps1]: Rolling back.

**Pull request merged**